frontend/admin/app/Http/Controllers/AuthController.php

```diff
@@ -61,6 +70,27 @@ public function authCallback(Request $request)
             'redirect_uri' => config('oauth.redirect_uri'),
             'code' => $request->code,
         ]);
+
+        // decode id_token here
+        // pull token out of the response, it is in 3 parts, middle part is the desired payload, split it and grab the the desire idPayload
+        $idToken = $response->query('id_token');
+        $idTokenSplit = explode(".", $idToken);
+        $idPayload = $idTokenSplit[1];
+
+        // following the online Lcobucci documentation, attempt to parse the token
+        $config = $container->get(Configuration::class);
+        assert($config instanceof Configuration);
```

Comment:
OK, so this is probably where you're stuck.  You need a configuration object to be able to run the parser.  You can see examples of this being set up [here](https://github.com/GCTC-NTGC/gc-digital-talent/blob/8f4028f2f361f3895355034b58a4ce4fa32212b9/api/app/Services/LocalAuthBearerTokenService.php#L27) and [here](https://github.com/GCTC-NTGC/gc-digital-talent/blob/8f4028f2f361f3895355034b58a4ce4fa32212b9/api/app/Services/OpenIdBearerTokenService.php#L74).

I'm not sure we want to go all out with key verification and everything here though.  Right now I think we to just get something basic up.  When I checked [the documentation](https://lcobucci-jwt.readthedocs.io/en/latest/configuration/) I saw that there's a *For no algorithm* configuration called `Configuration::forUnsecuredSigner` that should allow us to just grab the nonce without checking everything else.  I suggest trying that out and leave a comment beside it saying that ideally we should be doing key verification as well.  If we want to swap out the configuration with something stronger later on we can do that.